optuna/integration/pytorch_lightning.py

```diff
@@ -17,8 +17,8 @@
 class PyTorchLightningPruningCallback(Callback):
     """PyTorch Lightning callback to prune unpromising trials.
 
-    See `the example <https://github.com/optuna/optuna/blob/master/
-    examples/pytorch/pytorch_lightning_simple.py>`__
+    See `the example <https://github.com/optuna/optuna-examples/blob/
+    pytorch/pytorch_lightning_simple.py>`__
```

Comment:
The link is broken. I guess the branch name is missing.
```suggestion
    main/pytorch/pytorch_lightning_simple.py>`__
```